### DISCUSSION TITLE
Tailwind issue when using Docker + React

### DISCUSSION THREAD

**Discussion by weehong-1** [tailwindlabs/tailwindcss#3739] [Help]
When my docker runs npm ci, it shows package.json and the lock does not sync because of the TailwindCSS issue.

<img width="866" alt="PackageNotSync" src="https://user-images.githubusercontent.com/6753022/110263054-7e227c80-7ff0-11eb-8e3a-b2a94fb32b36.png">

But when I change the Docker command to npm install, it shows `(node:24) UnhandledPromiseRejectionWarning: Error: Cannot find module 'tailwindcss'`.

I have tried to change the package shown below and still receiving an error.
```
npm install -D tailwindcss@npm:@tailwindcss/postcss7-compat @tailwindcss/postcss7-compat postcss@^7 autoprefixer@^9

TO

npm install -D tailwindcss@latest postcss@latest autoprefixer@latest
```
Refer to the image name PostCSS8 and Tailwind@PostCSS7-Compact for the actual error message.

I have tried to move the tailwindcss from devDependencies to dependencies and that does not help either.

<img width="866" alt="PostCSS8" src="https://user-images.githubusercontent.com/6753022/110263065-84b0f400-7ff0-11eb-87b2-f99c8865df46.png">

<img width="866" alt="Tailwind@PostCSS7-Compact" src="https://user-images.githubusercontent.com/6753022/110263074-8a0e3e80-7ff0-11eb-99d6-168fe38ab68a.png">

**Comment by gauravprwl14** [comment]
I am also facing the same issue when running the app using the docker container.

 @WeeHong @bradlc is there update on the issue ??

**Reply by weehong-1** [reply]
Nope. I avoid using this actually. 😅
